### PR TITLE
Rollback changes from DST

### PR DESCRIPTION
**Pull request by mitogh**
**Ticket** https://central.tri.be/issues/99537

Introduced in 8f5e431bed24d6e2a72457f8504cadd5c70138bc

### CODE REVIEW

**Comment by mitogh** [comment]
Not sure if we need to create a PR in TEC as well @barryhughes ?

Thanks

File: readme.txt [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 == Changelog ==
 
+= [4.7.7.1] TBD =
+
+* Fix - Rollback changes introduced in commit #8f5e431 toa allow month view to render correctly.
```
Comment (barryhughes) [inline readme.txt]: - Let's swap out `commit #8f5e431` with `4.7.7`  (easier for normal users to understand)
- Correct `toa` to `to`

**Comment by barryhughes** [comment]
@mitogh - yep once this merges, let's update the common hash in TEC (same branch name).

**Comment by mitogh** [comment]
Changes has been applied. 👍

**Comment by barryhughes** [comment]
🏇

**Pull request merged**